Library/Homebrew/dev-cmd/mirror.rb

```diff
@@ -49,14 +43,18 @@ def mirror
 
       filename = downloader.basename
 
-      destination_url = "https://dl.bintray.com/homebrew/mirror/#{filename}"
+      destination_url = "https://dl.bintray.com/#{bintray_org}/#{bintray_repo}/#{filename}"
       ohai "Uploading to #{destination_url}"
 
-      content_url =
-        "https://api.bintray.com/content/homebrew/mirror/#{bintray_package}/#{f.pkg_version}/#{filename}?publish=1"
-      curl "--silent", "--fail", "--user", "#{bintray_user}:#{bintray_key}",
-           "--upload-file", downloader.cached_location, content_url
-      puts
+      bintray.upload(
+        downloader.cached_location,
+        repo:        bintray_repo,
+        package:     bintray_package,
+        version:     f.pkg_version,
+        sha256:      f.send(:stable).checksum,
```

Comment:
```suggestion
        sha256:      f.stable.checksum,
```


Technically, this should either use `active_spec` (though it's a protected method) or we use `f.stable.downloader` instead (and something else for `f.pkg_version`). In practice, it doesn't actually matter for Homebrew/core though.